### PR TITLE
Document using sandbox-pod

### PR DESCRIPTION
**Pull request by teobugslayer**

### CODE REVIEW

**Comment by teobugslayer** [comment]
ping @ErjanGavalji @rosen-vladimirov

**Comment by rosen-vladimirov** [comment]
:+1:

File: plugins/cocoapods.md [hunk 1]
```diff
@@ -104,6 +104,10 @@ This modifies the `MYCocoaPodsApp.xcodeproj` and creates a workspace with the sa
 
 > **IMPORTANT:** You will no longer be able to run the `xcodeproj` alone. NativeScript CLI will build the newly created workspace and produce the correct package.
 
+> **IMPORTANT:** NativeScript uses the `sandbox-pod` executable. This may prevent some pods from installing correctly. If you encounter such case, you may switch to the regular `pod`executable.
```
Comment (ikoevska) [inline plugins/cocoapods.md]:
```
If you encounter such cases, you can switch to the regular `pod` executable:
>
> 1. Open the NativeScript CLI configuration file, usually located in `/usr/local/lib/node_modules/nativescript/config/config.json`.
> 1. Change the value of `USE_POD_SANDBOX` to `false`.
```

**Comment by bg-k** [comment]
Thanks @teobugslayer , you saved my life

**Pull request merged**